katas/metal-slug/kevin/CharacterStatus.test.js

```diff
@@ -0,0 +1,9 @@
+import { CharacterStatus } from './CharacterStatus.js';
+test('Verify that getLifePoints get the life points of a soldier in a Tank', () => {
+    const setcharacterStatus = new CharacterStatus();
+    expect(setcharacterStatus.getLifePoints(1,3)).toBe(4);
```

Comment:
Try to avoid magic numbers when sending arguments for a function. `1` and `3` do not have meaning by themselves, you can improve that by putting them in a variable with meaningful name and then send them as arguments.